augur/api/view/init.py

```diff
@@ -91,4 +91,4 @@ def write_settings(current_settings):
 # Initialize logging
 def init_logging():
     global logger
-    logger = AugurLogger("augur_view", reset_logfiles=True).get_logger()
+    logger = AugurLogger("augur_view", reset_logfiles=False).get_logger()
```

Comment:
Does this only affect view?

Reply:
No, it previously would reset all log files globally. So now the log files will continue to exist after starting Augur